src/components/accessibility/Listbox.js

```diff
@@ -74,6 +76,7 @@ class Listbox extends React.Component {
         break;
       case 'enter':
       case 'space':
+        if (this.props.withSearch && this.state.focusedIndex <= 0) return
```

Comment:
Nitpick but if the search input will always be index 0 then can you just `this.state.focusedIndex === 0`? Also, can you add a comment here that explains why you're bailing out on preventing default on space?

Reply:
@cerinman
I observed two values the focusedIndex state could be when SearchInput is focused: -1 and 0. So this.state.focusedIndex === 0 will not work for every case.

I will add a comment to document why this line exists.

Reply:
sounds good.